src/lib/services/proxy-service.ts

```diff
@@ -99,14 +100,13 @@ export class ProxyService {
     private async clientForProxyToken(token: ApiUser): Promise<Unleash> {
```

Comment:
This is currently unchanged and I was lucky it was that way because it fits well with the change I did. 
The reason for this is probably that we need to execute a `client.start()` [here](https://github.com/Unleash/unleash/blob/217e4a491798dbfa343d623c6c547f39a7fbb999/src/lib/services/proxy-service.ts#L112-L132) and that's async, which bubbles up until this line